modules/graceful-shutdown.adoc

```diff
@@ -65,13 +65,20 @@ node/ci-ln-mgdnf4b-72292-n547t-worker-c-vcmtn cordoned
 $ for node in $(oc get nodes -l node-role.kubernetes.io/worker -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm drain ${node} --delete-emptydir-data --ignore-daemonsets=true --timeout=15s --force ; done
 ----
 
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider’s web console, or by running the following loop:
+. Shut down all of the nodes in the cluster. You can do this from your cloud provider’s web console, or by running the following loop. Shutting down the nodes by using one of these methods allows pods to terminate gracefully, which reduces the chance for data corruption.
++
+[NOTE]
+====
+Ensure that the control-plane node with the API VIP assigned is the last node processed in the loop. Otherwise, the shutdown command fails.
+====
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done <1>
 ----
 +
+<1> `-h 1` indicates how long, in minutes, this process lasts before the control-plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to `-h 10` or longer to make sure all the compute nodes have time to shut down first. 
```

Comment:
```suggestion
<1> `-h 1` indicates how long, in minutes, this process lasts before the control plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to `-h 10` or longer to make sure all the compute nodes have time to shut down first. 
```

Reply:
Per the RHSSG, control plane doesn't get a hyphen: https://redhat-documentation.github.io/supplementary-style-guide/#control-plane